test/sql/hnsw_dist_func.sql

```diff
@@ -0,0 +1,133 @@
+CREATE TABLE small_world_l2 (
+    id varchar(3),
+    vector real[]
```

Comment:
Can I create a table with `integer[]` column and have l2sq distanced index on it?
Let's have a test for this.

Reply:
no, we should add operator class for `integer[]` type in order to work.